orca_detector/inference.py

```diff
@@ -26,21 +27,28 @@
 RUN_TIMESTAMP = datetime.datetime.now().isoformat('-')
 
 
-def create_network(num_classes, weights_path):
+def create_network(model_name, num_classes, weights_path):
     """ 
     Instantiate trained model from given weights.
     Create the output shape based on num_classes.
     """
 
-    sound_extractor = OrcaVGGish(load_weights=True,
-                                 weights=weights_path,
-                                 out_dim=num_classes,
-                                 pooling='avg').get_model()
-
+    if model_name == 'vggish':
+        sound_extractor = OrcaVGGish(load_weights=True,
```

Comment:
Why do we call this sound_extractor?

Reply:
because it was branch copied from train.py.  I can rename.

Reply:
fixed (in both places)